enclave/core/windows/reloc.c

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
```

Comment:
How does this link without a definition for `oe_call_init_functions` or `oe_call_fini_functions` ? Or is that future work?

Reply:
future work.